chains.yaml

```diff
@@ -12,6 +12,15 @@
   binaries:
     - /go/bin/akash
 
+# Assetmantle
+- name: assetmantle
+  github-organization: assetmantle
+  github-repo: node
+  language: go
+  build-target: build-linux
```

Comment:
Let's use `install` as the `build-target` here so that it will also build for arm64. In the assetmantle node Makefile, they hardcode to amd64 as the arch for the `build-linux` target: https://github.com/AssetMantle/node/blob/5b2b0dcb37b107b0e0c1eaf9e907aa9f1a1992d9/Makefile#L103

With that, bin can be sourced from `/go/bin/`

Reply:
We're not planning for `arm64` support for now. And makefile is not yet stable with `install` target